source/DataAccess/Oracle/OracleDDLHelper.ts

```diff
@@ -0,0 +1,26 @@
+import IDDLHelper from '../IDDLHelper';
+import { injectable } from 'inversify';
+
+@injectable()
+export default class OracleDDLHelper implements IDDLHelper {
```

Comment:
I have a feeling this will change as we discover the need to be smarter about searching for tables - specifically filtering by schema/owner - the plan being to allow that information to vary through the constructor and keep the getDDLQuery signature stable.

Reply:
I wasn't exactly sure of the class responsibility from the class name. Maybe something like OracleDDLStatementFactory or something similar? I'm not sure that's any more clear, so I don't really have better suggestions here.